.rubocop.yml

```diff
@@ -61,7 +59,7 @@ Style/StringChars: # (new in 1.12)
 Style/SwapValues: # (new in 1.1)
   Enabled: true
 
-Gemspec/DateAssignment: # (new in 1.10)
+Gemspec/DeprecatedAttributeAssignment: # (new in 1.10)
```

Comment:
Why this line is change? Could you put changes related to rubocop configuration on a single commit or a separate PR without including slurm change? (Or we can simply merge this PR first: https://github.com/aws/aws-parallelcluster-cookbook/pull/1571)

Reply:
Sorry, I thought I never staged this file. I made this change following my local removal of gitlab-security and encountering an error that suggested this change.